processors/src/ClusterOnTrackAnaProcessor.cxx

```diff
@@ -1,6 +1,6 @@
 #include "ClusterOnTrackAnaProcessor.h"
 #include "TBranch.h"
-
+#include "Collections.h"
```

Comment:
Try removing this. We might need to pass in some config params to be able to do that, but you can see examples of how this is done in other processors.